lib/components/inputs/Select/Select.tsx

```diff
@@ -244,7 +244,7 @@ function Select(props: SelectProps) {
     'select-wrapper-is-multi': isMulti,
     'has-error': error,
     'no-label': !label,
-    'expand-input-on-focus': expandInputOnFocus
+    'expand-input-on-focus': expandInputOnFocus ?? isMulti
```

Comment:
I guess we also wanted to have it by default in the TagsBox?

Reply:
yes, please do

Reply:
Done